db/migrations/0005.change_pks_to_canvas_id.py

```diff
@@ -0,0 +1,79 @@
+#
+# file: migrations/0005.change_pks_to_canvas_id.py
+#
+from yoyo import step
+
+__depends__ = {'0004.add_sis_id'}
+
+steps = [
+    # Remove foreign key constraints
+    step('''
+        ALTER TABLE enrollment
+        DROP FOREIGN KEY enrollment_ibfk_1
```

Comment:
What is this `enrollment_ibfk_1`?

Reply:
Those are the names of the foreign key constraints. Those were automatically created by the foreign key processes in the first migration. You can see them by running a command, let me find it.

Reply:
If you just run `SHOW CREATE TABLE enrollment` in `course_inventory_test`, you'll see the constraint names.

Reply:
`ib` must stand for `InnoDB`, the engine we're using?